examples/campaign_management/create_campaign_experiment.py

```diff
@@ -0,0 +1,107 @@
+            'CampaignExperimentService', version='v1')
+
+        # A Long Running Operation (LRO) is returned from this
+        # asynchronous request by the API.
+        campaign_experiment_lro = (
+            campaign_experiment_service.create_campaign_experiment(
+                customer_id, campaign_experiment))
+    except GoogleAdsException as ex:
+        print('Request with ID "{}" failed with status "{}" and includes the '
+              'following errors:'.format(ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "{}".'.format(error.message))
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: {}'.format(field_path_element.field_name))
+        sys.exit(1)
+
+    print('Asynchronous request to create campaign experiment with '
+          'resource name "{}" started.'.format(
+              campaign_experiment_lro.metadata.campaign_experiment))
+    print('Waiting until operation completes.')
+
+    # Poll until the operation completes.
+    campaign_experiment_lro.result()
+
+    # Retrieve the campaign experiment that has been created.
+    ga_service = client.get_service('GoogleAdsService', version='v1')
+    query = (
+        'SELECT campaign_experiment.experiment_campaign '
+        'FROM campaign_experiment '
+        'WHERE campaign_experiment.resource_name = "{}"'.format(
+            campaign_experiment_lro.metadata.campaign_experiment))
+
+    results = ga_service.search(customer_id, query=query, page_size=1)
+
+    try:
+        for row in results:
+            print('Experiment campaign "{}" finished creating.'.format(
+                row.campaign_experiment.experiment_campaign.value))
+    except GoogleAdsException as ex:
+        print('Request with ID "{}" failed with status "{}" and includes the '
+              'following errors:'.format(ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "{}".'.format(error.message))
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: {}'.format(field_path_element.field_name))
+        sys.exit(1)
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description=('Create a campaign experiment based on a campaign draft.'))
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument('-c', '--customer_id', type=six.text_type,
+                        required=True, help='The Google Ads customer ID.')
+    parser.add_argument('-i', '--campaign_draft_resource_name',
```

Comment:
why -i? since it is a name, I'd suggest -n

Reply:
Done